openllm-core/src/openllm_core/config/configuration_qwen.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import openllm_core
+
+class QwenConfig(openllm_core.LLMConfig):
+  """Qwen-7B is the 7B-parameter version of the large language model series, Qwen (abbr. Tongyi Qianwen),
+  proposed by Alibaba Cloud. Qwen-14B is a Transformer-based large language model,
+  which is pretrained on a large volume of data, including web texts, books, codes, etc.
+  Additionally, based on the pretrained Qwen-14B, we release Qwen-14B-Chat, a large-model-based AI assistant,
+  which is trained with alignment techniques.
+  Refer to [Qwen's GitHub page](https://github.com/QwenLM/Qwen) for more information.
+  """
+
+  __config__ = {
+    'name_type': 'lowercase',
+    'trust_remote_code': True,
+    'timeout': 3600000,
+    'url': 'https://github.com/QwenLM/Qwen',
+    'requirements': ['cpm-kernels'],
```

Comment:
Seems like it also requires `tiktoken`
```suggestion
    'requirements': ['cpm-kernels', 'tiktoken'],
```